animalfacts.py

```diff
@@ -329,6 +329,7 @@ def check_comment_for_animal(comment, reddit):
     botengine('pig', '\spigs?\s', reddit, PIG_FACTS, comment)
     botengine('pigeon', '\spigeons?\s', reddit, PIGEON_FACTS, comment)
     botengine('platypus', '\splatypuse?s?\s', reddit, PLATYPUS_FACTS, comment)
+    botengine('prairie dog', '\sprairie dogs?s?\s', reddit, PRAIRIEDOG_FACTS, comment)
```

Comment:
Another one for #168 @joel-g so we can't really take this yet. Also do you want the tuple name as two words? I am so picky!